### PR TITLE
[Issue 374] Trim day

### PR DESCRIPTION
**Pull request by gkovats**
Stop that whitespace from triggering a meeting to appear as an appointment.

### CODE REVIEW

File: src/helpers/translate-google-sheet.ts [hunk 1]
```diff
@@ -104,6 +104,7 @@ export function translateGoogleSheet(
     }
 
     if (meeting.day && typeof meeting.day === 'string') {
+      meeting.day = meeting.day.trim();
```
Comment (joshreisner) [inline src/helpers/translate-google-sheet.ts:107]: could we just trim every value?

Comment (gkovats) [reply src/helpers/translate-google-sheet.ts:107]: _Totally missed_ this. Will update.

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,7 @@ npm-debug.log
 coverage
 test-results/
 playwright-report/
+public/custom
```
Comment (gkovats) [inline .gitignore:7]: @joshreisner  I have some demo app test pages locally, they're looking at junk google sheets on my personal account. Wanted a local directory I can save work to.

File: package.json [hunk 3]
```diff
@@ -7,7 +7,9 @@
     "test": "jest",
     "test-watch": "jest --watch",
     "test-coverage": "jest --coverage",
-    "serve": "serve public"
+    "serve": "serve public",
+    "watch": "npx mix watch",
+    "build": "npx mix --production"
```
Comment (gkovats) [inline package.json:12]: Just adding the 2 commands we run locally to stand up the server.

File: src/helpers/translate-google-sheet.ts [hunk 4]
```diff
@@ -52,6 +52,8 @@ export function translateGoogleSheet(
     if (!row.filter(e => e).length) return;
 
     const meeting: JSONData = {};
+    // trim cells
+    row = row.map(cell => String(cell).trim());
```
Comment (gkovats) [inline src/helpers/translate-google-sheet.ts:57]: Moved to trimming every row cell

**Review by joshreisner — APPROVED** [review]

**Pull request merged**